js/usercss.js

```diff
@@ -195,11 +196,40 @@ var usercss = (() => {
         parseJSONValue(state);
         state.errorPrefix = '';
         if (Array.isArray(state.value)) {
-          result.options = state.value.map(text => createOption(text));
+          result.options = state.value.map(text => {
+            if (text.endsWith('*')) {
+              text = text.replace(/\*$/, '');
+              deflt = text;
+            }
+            return createOption(text);
+          });
+        } else {
+          result.options = Object.keys(state.value).map(k => {
+            if (k.endsWith('*')) {
+              state.value[k] = state.value[k].replace(/\*$/, '');
+              k = k.replace(/\*$/, '');
```

Comment:
use `.slice(0, -1).trim()` in both cases

Reply:
er, I guess trim() isn't needed because some values may really need those trailing spaces